deepchem/models/tests/test_layers.py

```diff
@@ -1450,3 +1450,22 @@ def test_torch_highway_layer():
     output_tensor = highway_layer(input_tensor)
 
     assert output_tensor.shape == (batch_size, feat_dim)
+
+ 
+@pytest.mark.torch
+def test_torch_attn_lstm_embedding():
```

Comment:
@Dragonwagon18 please add a value test also, the test that you added tests only the shape

Reply:
added